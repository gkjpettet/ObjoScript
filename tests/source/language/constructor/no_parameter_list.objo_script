class Foo {
  constructor { // expect error
    System.print("ok")
  }
}