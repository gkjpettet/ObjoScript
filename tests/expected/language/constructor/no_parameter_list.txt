Expected an opening parenthesis after the `constructor` keyword.